### PR TITLE
Quiz background

### PR DESCRIPTION
**Pull request by A-Ghoujal**

### CODE REVIEW

**Review by YunusEmreBEYAZ — APPROVED** [review]

**Review by dakravchenko — APPROVED** [review]

**Review by sima-milli — COMMENTED** [review]
Can you please move img folder inside public folder

File: src/data.js [hunk 1]
```diff
@@ -66,86 +66,89 @@ export const quizData = {
   currentQuestionIndex: 0,
   questions: [
     {
-      text: ' question 1 ',
+      text: ' What is the brightest planet in our solar system? ',
```
Comment (sima-milli) [inline src/data.js]: if this file is the same as @dakravchenko's PR, can you revert it please

File: public/style.css [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 /* add the img folder with this img 'stars.jpg'  */
 body {
-  background: url('img/stars.jpg') repeat 0 0;
+  background: url(../public/img/stars.jpg) repeat 0 0;
```
Comment (sima-milli) [inline public/style.css]: this url should be wrapped in double quotes `""` https://www.w3schools.com/cssref/css3_pr_background.php

**Review by sima-milli — APPROVED** [review]
I fixed some conflicts with master, this should work now

**Pull request merged**